File: Modul6/Python/PRAK603-2310817210003-MuhammadAzrianzan.py
n1, n2 = input("").split()
n1 = int(n1)
n2 = int (n2)
if n1 == n2:
    inputan = 2 * n1
    myList = list(map(int, input().split()))
    myList2 = []
    for i in range(n1):
        hasil = myList[i] * myList[n2]
        n2 = n2 + 1
        myList2.append(hasil)
    for j in range(n1):
        print(myList2[j], end=" ")
else:
    print("Jumlah tidak sama")

File: Modul6/C/PRAK603-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    int n1, n2;
    scanf("%d %d", &n1, & n2);
    if (n1 == n2) {
        int inputan = 2 * n1;
        int myArray[inputan];
        int myArray2[inputan];
        for (int i = 0; i < inputan; i++) {
            scanf("%d", &myArray[i]);
        }
        for (int j = 0; j < n1; j++) {
            int hasil = myArray[j] * myArray[n2];
            n2 = n2 + 1;
            myArray2[j] = hasil;
        }
        for (int x = 0; x < n1; x++) {
        printf("%d ", myArray2[x]);
    }
    }
    else {
        printf("Jumlah tidak sama");
    }
    return 0;
}